Cue/app/tabs/library/LibraryHome.android.js

```diff
@@ -0,0 +1,160 @@
+// @flow
+
+import React from 'react'
+import { View, Text, Image, TouchableOpacity, Navigator, Modal } from 'react-native'
+
+import { connect } from 'react-redux'
+
+import type { Deck } from '../../api/types'
+
+import CueColors from '../../common/CueColors'
+import CueHeader from '../../common/CueHeader'
+import CueIcons from '../../common/CueIcons'
+import { MKButton } from 'react-native-material-kit'
+
+import prompt from 'react-native-prompt-android';
+import LibraryListView from './LibraryListView'
+import { createDeck, loadLibrary, syncLibrary } from '../../actions/library'
+
+const styles = {
+  container: {
+    flex: 1,
+    backgroundColor: 'white'
+  },
+  bodyContainer: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  fabContainer: {
+    position: 'absolute',
+    bottom: 16,
+    right: 16,
+  },
+  fab: {
+    width: 56,
+    height: 56,
+    borderRadius: 28,
+    marginTop: 16,
+  },
+  fabIcon: {
+    tintColor: 'white',
+  },
+}
+
+
+type Props = {
+  decks: Array<Deck>,
+  navigator: Navigator,
+  onPressMenu?: () => void,
+  localChanges: {}
+}
+
+class LibraryHome extends React.Component {
+  props: Props
+
+  state: {
+    refreshing: boolean
+  }
+
+  _refresh() {
+    this.setState({
+      ...this.state,
+      refreshing: true
+   });
+   this.props.onSyncLibrary(this.props.localChanges).then(failedSyncs =>{
+     //TODO: issue #65
+     // will need promise to return serverDeck version of the failedSync
+   })
+  }
+
+  componentWillReceiveProps(newProps: Props) {
+    this.setState({
+      ...this.state,
+      refreshing: false
+    });
+  }
+
+  constructor(props: Props) {
+   super(props);
+   this.state = {
+     refreshing: false
+   };
+   this._refresh = this._refresh.bind(this)
+  }
+
+  render() {
+    let leftItem = {
+      title: 'Menu',
+      icon: CueIcons.menu,
+      onPress: this.props.onPressMenu
+    }
+
+    let overflowItems = [{
+        key: 'Select',
+        title: 'Select'
+    }]
+
+    const AddFAB = MKButton.coloredFab()
+      .withBackgroundColor(CueColors.primaryTint)
+      .withStyle(styles.fab)
+      .withOnPress(() => {
+        prompt(
+          'Create New Deck',
```

Comment:
Extremely minor nitpick: According to Android / Material conventions, this should be in sentence case, i.e., `Create new deck`. This is wrong in the UX mock, sorry.